### PR TITLE
docs: add Your First WunderGraph Application tutorial

### PR DESCRIPTION
**Pull request by fiam**
#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md [hunk 1]
```diff
@@ -3,3 +3,209 @@ title: Your First WunderGraph Application
 pageTitle: WunderGraph - Your First WunderGraph Application
 description:
 ---
+
+In this tutorial we're going to guide you through a a very simple
+WunderGraph application, and we'll make it more complex later.
+
+WunderGraph applications are just Node packages, so if you're already familiar with
```
Comment (jensneuse) [inline docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md:10]: I'm not sure I like the wording here.

File: docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md [hunk 2]
```diff
@@ -3,3 +3,209 @@ title: Your First WunderGraph Application
...
+
+## Create and initialize your new application
+
+For this tutorial, we'll use the WunderGraph `application` template,
+to save us some typing.
+
+```shell
+# Init a new project with the application template
+npx -y @wundergraph/wunderctl init \
+	--template application -o countries
```
Comment (jensneuse) [inline docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md:27]: I think if you're using the application template, you can omit `--template application`

File: docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md [hunk 3]
```diff
@@ -3,3 +3,209 @@ title: Your First WunderGraph Application
...
+import {
+  Application,
+  configureWunderGraphApplication,
+  cors,
+  introspect,
+} from '@wundergraph/sdk'
+
+const continentsAPI = introspect.graphql({
+  // Namespace inside our virtual graph
+  apiNamespace: 'the_continents',
```
Comment (jensneuse) [inline docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md:114]: Why continents? it's the countries API?
I'd stay consistent and name the api "countries" as well as the namespace.
I'd also explain what a namespace is and why it's important. (At least briefly)

Also, please don't suggest namespaces like "the_xxx".
Namespaces should be [a-z].

File: docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md [hunk 4]
```diff
@@ -3,3 +3,209 @@ title: Your First WunderGraph Application
...
+})
+
+const myApplication = new Application({
+  name: 'app',
+  apis: [continentsAPI],
+})
+
+configureWunderGraphApplication({
+  application: myApplication,
+  cors: {
```
Comment (jensneuse) [inline docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md:126]: Actually, I'd suggest that you change the API here.
Can you make cors settings optional and just allow anything?
We can later use the CI to validate this config and warn the user if they use "allow all" in production.

**Review by jensneuse — CHANGES_REQUESTED** [review]

File: docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md (hunk 2 above)
Comment (fiam) [reply docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md:27]: You're right, let's drop that

File: docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md (hunk 3 above)
Comment (fiam) [reply docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md:114]: The idea was to make it distinct from the `continents` in the graphql API. Having `continents_continents` makes telling apart what's namespace and what comes from the API a bit confusing. What do you think about changing the namespace to `world`?

File: docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md (hunk 4 above)
Comment (fiam) [reply docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md:126]: I agree. PR for the API change is in https://github.com/wundergraph/wundergraph/pull/270 - I will rewrite the doc assuming we'll merge it.

File: docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md (hunk 1 above)
Comment (fiam) [reply docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md:10]: What about...

- `At their heart, WunderGraph applications are Node packages`
- `WunderGraph applications are also Node packages`

File: docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md (hunk 3 above)
Comment (jensneuse) [reply docs-website/src/pages/docs/tutorials/your-first-wundergraph-application.md]: world is good

**Review by jensneuse — APPROVED** [review]

**Comment by fiam** [comment]
@jensneuse This is running into the problem with `build_test` not triggering. Could you please override and merge? Thanks!

**Pull request merged**